backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveDiskCommand.java

```diff
@@ -170,11 +170,12 @@ protected Map<String, Pair<String, String>> getExclusiveLocks() {
 
     private String getDiskIsBeingMigratedMessage() {
         if (cachedDiskIsBeingMigratedMessage == null) {
+            String diskName = getParameters().getNewAlias() != null ? getParameters().getNewAlias() :
+                    getParameters().getDiskAlias() != null ? getParameters().getDiskAlias() : "";
```

Comment:
@bennyz is it the right way to take the disk alias from the parameters in this flow or should we use the alias of the disk we get from the database?

Reply:
yes, I tried to look around but I'm not sure where/if it is set for MoveDisk